### PR TITLE
qa/tasks: add remaining tests for fs volume

### PR DESCRIPTION
**Pull request by joscollin**
Fixes: https://tracker.ceph.com/issues/42872

### CODE REVIEW

File: qa/tasks/cephfs/test_volumes.py [hunk 1]
```diff
@@ -117,7 +122,53 @@ def test_connection_expiration(self):
         # Now wait for the mgr to expire the connection:
         self.wait_until_evicted(sessions[0]['id'], timeout=90)
 
+    def test_volume_create(self):
+        """
+        That the volume can be created and then cleans up
+        """
+        volname = self._generate_random_volume_name()
+        self._fs_cmd("volume", "create", volname)
```
Comment (batrick) [inline qa/tasks/cephfs/test_volumes.py]: Have you tried testing this? I would expect a failure because multifs is not gated off with an experimental flag.

Comment (joscollin) [reply qa/tasks/cephfs/test_volumes.py]: Yes, Please see [1] and [2]. It's already there.

[1] https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/test_volumes.py#L45
[2] https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/test_volumes.py#L97

**Comment by joscollin** [comment]
retest this please

**Comment by batrick** [comment]
Please do a teuthology run and link the results.

**Comment by joscollin** [comment]
http://pulpito.front.sepia.ceph.com/jcollin-2019-12-12_08:40:18-fs-wip-B42872-testing-distro-basic-smithi/

```
2019-12-12T10:04:44.157 INFO:tasks.cephfs_test_runner:test_volume_create (tasks.cephfs.test_volumes.TestVolumes) ... ok
2019-12-12T10:05:52.273 INFO:tasks.cephfs_test_runner:test_volume_ls (tasks.cephfs.test_volumes.TestVolumes) ... ok
2019-12-12T10:06:47.219 INFO:tasks.cephfs_test_runner:test_volume_rm (tasks.cephfs.test_volumes.TestVolumes) ... ok
```
Looks Good.

**Pull request merged**